token-syncer/src/token_syncer/utils.clj

```diff
@@ -13,9 +13,27 @@
 ;; See the License for the specific language governing permissions and
 ;; limitations under the License.
 ;;
-(ns token-syncer.utils)
+(ns token-syncer.utils
+  (:require [clj-time.coerce :as tc]
+            [clj-time.core :as t]
+            [clj-time.format :as f]))
 
 (defn successful?
   "Returns true if the response has a 2XX status code."
   [{:keys [status]}]
   (and (integer? status) (<= 200 status 299)))
+
+(defn iso8601->millis
+  "Convert the ISO 8601 string to numeric milliseconds."
+  [date-str]
+  (-> (:date-time f/formatters)
```

Comment:
Nitpick: These threading calls are awkward.  For me, threading makes the most sense when you start with input data and go through a series of transformations.

Reply:
Changed.